src/libsync/propagateupload.cpp

```diff
@@ -208,10 +218,12 @@ void PropagateUploadFileCommon::slotComputeContentChecksum()
         return;
     }
 
-    const QString filePath = propagator()->getFilePath(_item->_file);
+    const QString filePath = propagator()->getFilePath(_fileToUpload._file);
```

Comment:
That's `_fileToUpload._path`
Please use that everywhere instead of re-computing it.